### DISCUSSION TITLE
Optional screen refresh option when calling a function

### DISCUSSION THREAD

**Discussion by lbaraldi** [VasilKalchev/LiquidMenu#64] [Ideas]
**Is your feature request related to a problem? Please describe.**
i have a 4-key i2c keyboard. Two buttons to change the focus and the other two to enter, exit, increase or decrease a variable.
All manipulation is done through callback functions.

I implemented a functionality on the keyboard that when a key is held down, then the function is called repeatedly in order to change the variable value continuously until reaching the desired value.

the problem is that the LCD display starts to flicker badly

**Describe the solution you'd like**
**It would be interesting if:**
  - When attaching a function to a line, we could pass the update type option to that function.

  Example:
- Soft update
LiquidLine :: attach_function(number, function, **_false_**)

- "Standard" update
  LiquidLine :: attach_function(number, function)

**Describe alternatives you've considered**
**Another option would be to choose when the function call is made:**
LiquidMenu :: call_function(number, **_false_**)

**Additional context**
That way it would be easy to keep updating the LCD display without flickering... 

Example:
When incrementing a variable, we check inside the function that makes the increment if its resulting value has the remainder of the division by 10 equal to zero!

```
.
.
.
if (...) {
  int i = i++;
  if ( (i % 10) == 0) {
    menu.update();
  }
. 
.
. 
}
```

**Comment by VasilKalchev** [comment]
Hello lbaraldi.

I added an optional argument to the `call_function` method:
https://github.com/VaSe7u/LiquidMenu/blob/c19f203d6467bb3c4bc46605451fee981007f976/src/LiquidMenu.h#L992
